tests: TestCheckPortAvailability fails when port 9000 busy.

## Expected behaviour

Tests to pass irrespective whether port 9000 is busy. 
## Actual behaviour

Fails when 9000 is busy.
--- FAIL: TestCheckPortAvailability (0.00s)
        checkport_test.go:37: checkPortAvailability test failed for port: 9000. Error: listen tcp :9000: bind: address already in use
## Steps to reproduce the behaviour

Run any server on 9000 and do `go test` on Minio repo root. 
## Minio version

Master at b23605a2b5397b0303f4ab1ebe3111c93c6796eb
## System information

Able to produce on both Mac and Linux.

--- a/checkport_test.go
+++ b/checkport_test.go
@@ -23,12 +23,13 @@ import (
 	"testing"
 )
 
+// Tests for port availability logic written for server startup sequence.
 func TestCheckPortAvailability(t *testing.T) {
 	tests := []struct {
 		port int
 	}{
-		{9000},
-		{10000},
+		{getFreePort()},
+		{getFreePort()},
 	}
 	for _, test := range tests {
 		// This test should pass if the ports are available

